book/src/SUMMARY.md

```diff
@@ -32,6 +32,8 @@
   - [JSON RPC API](jsonrpc-api.md)
   - [JavaScript API](javascript-api.md)
   - [solana-wallet CLI](wallet.md)
+- [Testing](testing.md)
+  - [Cluster Test Framework](cluster-test-framework.md)
```

Comment:
Per CONTRIBUTING.md, this should sit in the proposed changes section until implemented?

Reply:
It’s implemented already.

Reply:
Grepping "CTF" and "Framework" in the codebase indicates that isn't the case.

Reply:
Rename the LocalCluster to LocalClusterCTF?

Reply:
No, that wouldn't make any sense.  I would think there'd be a RemoteCluster that launches cloud nodes, and that integration tests would be coded against some `cluster_test_framework.rs`, and that they'd have a `dev-dependency` on a `local_cluster` that could be trivially replaced with a `remote_cluster` dependency to run the same test against a RemoteCluster.